src/Engine/Renderer/Mesh/Mesh.cpp

```diff
@@ -111,7 +111,7 @@ namespace Ra {
             GLenum type = GL_FLOAT;
 #endif
             constexpr GLuint size = VecArray::Vector::RowsAtCompileTime;
-            constexpr GLboolean normalized  = GL_FALSE;
+             GLboolean normalized  = GL_FALSE;

```

Comment:
This is due to a lack of constexpr in glbinding. In glbinding GLboolean is declared as a class (and not an unsigned int as in gl.h) with two static objects GL_TRUE and GL_FALSE. As their is no constructor declared as constexpr in glbinding/gl/boolean.h, we canot declare normalized as a constexpr here. An what is the benefit of declaring a local (temporery) variable as a constexpr ?